modules/lua-loader.nix

```diff
@@ -0,0 +1,35 @@
+{
+  config,
+  lib,
+  ...
+}:
+with lib; let
+  cfg = config.luaLoader;
+in {
+  options.luaLoader = {
+    enable = mkOption {
+      type = types.bool;
+      default = true;
```

Comment:
Shouldn't we set it to false by default ? As it is false right now, we want to forward neovim defaults right ?

Reply:
Indeed. Fixed !